### PR TITLE
Adds Normandy Frigate and Hellfire Gunship

### PR DESCRIPTION
**Pull request by LittleBuilderJane**
## About the PR

The TGNV Patrol Frigate "Normandy" and TGN Aerospace Gunship "Hellfire" are added into the game.
The Normandy has a fully stocked medical bay, CBURN armory and Ship Security barracks, as well as a fully functional Engineering.
The Hellfire has three Svalinn laser machine guns and is a smaller attack craft with atmos.
## Why / Balance

Added to be used in my Contagion event exclusively. (Though of course others are very much welcome to use these too.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
Normandy3
Normandy2
Normandy1
Hellfire
## Breaking changes

**Changelog**

CL removed - non player facing change
- add: Added the TGNV Patrol Frigate "Normandy" and it's associated Aerospace Gunship "Hellfire". The TerraGov Navy and Marine Corps make their debut for Events !

### CODE REVIEW

**Comment by Axionyxx** [comment]
If these are only for events, they should probably just be uploaded to the server whenever they're needed

**Comment by LittleBuilderJane** [comment]
> If these are only for events, they should probably just be uploaded to the server whenever they're needed

Uhhhh....how exactly would that be done ?
Is that even something anyone else other than you can do ?

**Comment by deltanedas** [comment]
`uploadfile` command iirc

**Comment by LittleBuilderJane** [comment]
> `uploadfile` command iirc

Yeah but Admins don't have perms to use that.

**Comment by IamVelcroboy** [comment]
**Normandy**
TLDR, make this closer to a standard shuttle even if it's admin only. Assume players can/will get their hands on it. And I'll quote Lyndoman, "the dream is to have them be cohesive with everything else"

- [ ] Way too many mats. Since this is an admin use shuttle, admin can spawn the mats they need at the time.
- [ ] Use fueled gens(can provide fuel) again admins can replace with debug gens when they use it
- [ ] Same with meds in medbay. Cut down on overall meds and limit nanomeds to like 2 walls and 1 standard
- [ ] Same with Biomass
- [ ] So many donks
- [ ] Jani closet is missing light
- [ ] All doors are CC but shuttle/airlock are NukeOps??
- [ ] Thematically we want to stick with plastitanium being used exclusively for syndicate structures. (Word on the street we might be getting some different walls for NT combat shuttles. So, stay tuned for that.)
- [ ] Keep to the standards of not running pipes under walls and color them
- [ ] Same for other underground entities (LV is kinda whack and there will be lots of crossover)
- [ ] Hallways could use some decals. Nothing excessive. Just some trim or something
- [ ] Avoid lights on windows 
image
- [ ] and doors 😛
image
- [ ] Ax cabinet is on the table 
image
- [ ] These thrusters won't work with tile in front of them(lattice is ok)
image
- [ ] Name electronics (APCs, SMESs, Subs)
- [ ] Oni can't pass this section 
image
- [ ] Replace cloning pods with metem pods
- [ ] Not quite sure why but beacons not showing up on the map, perhaps just delete/replace them 🤷‍♂️
- [ ] running cable under SMES breaks them 
image


This is fucking cool though 
image
Overall, the shuttle is very cool. Just stick to the same design standards as you would any other shuttle regardless of it being admin only.

**Comment by IamVelcroboy** [comment]
Will review the other shuttle later. Need to go to work for a while.

**Comment by IamVelcroboy** [comment]
**Hellfire**
- [ ] Is it meant to be a nukie shuttle?
image
- [ ] Remove the debug gen. ~~Use a plasma or uranium gen with a stack of fuel,~~(I guess there's no access to fuel them) or use a RTG and VV the power output to like 30k

All else looks good.

**Comment by LittleBuilderJane** [comment]
> **Hellfire**
> 
> * [ ]  Is it meant to be a nukie shuttle?
>   image
> * [ ]  Remove the debug gen. ~Use a plasma or uranium gen with a stack of fuel,~(I guess there's no access to fuel them) or use a RTG and VV the power output to like 30k
> 
> All else looks good.

The reason I use NukeOps external airlocks is because there are places that have exterior access...but aren't supposed to be accessed by Station Crew. The Normandy and Hellfire's crew have AA as they are supposed to be emergency responders. The ship uses Centcom doors because we DO NOT have Independent/Terra Gov doors....and we have no Centcom-locked external airlocks, so I use the Nukie ones because they look the exact same as normal ones. 

The amount of meds and mats is so that the crew can actually treat the crew after a Zombie outbreak and help repair their station, and the Plastitanium walls are because those walls feel the best suited for a TerraGov Navy Ship, over Shuttle Walls and Reinforced standard walls. It's the same reason for why it has cables go under walls and generators that don't need refueling, it's not meant to be crewed fully.

I'll fix all the other stuff you pointed out though, on both ships ! 😃

**Comment by TadJohnson00** [comment]
Is this still being worked on @LittleBuilderJane?

**Comment by deltanedas** [comment]
reviews need addressing, stale
